ui/curses_ui.py

```diff
@@ -371,7 +371,7 @@ def main_ui(stdscr):
                 entry_win.clear()       
                 # entry_win.refresh()
 
-        elif char == curses.KEY_BACKSPACE or char == 127:
+        elif char in (curses.KEY_BACKSPACE, 127):
```

Comment:
```suggestion
        elif char in (curses.KEY_BACKSPACE, chr(127)):
```
This seems to fix it on my end

Reply:
Okay cool, let's go with that then!